src/server/graphql/httpGraphQLHandler.js

```diff
@@ -1,8 +1,9 @@
-import Schema from './rootSchema';
+// import Schema from './rootSchema';
 import {graphql} from 'graphql';
 
 export default async (req, res) => {
-  // Check for admin privileges
+  // eslint-disable-next-line global-require
+  const Schema = require('./rootSchema');
```

Comment:
a dynamic require means no need to start the pool on bootup, which means faster boot times.